submitted_models/jhuapl_edge540xl_sensor_config_1/launch/vehicle_topics.launch

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0"?>
+<launch>
+  <env name="IGN_TRANSPORT_TOPIC_STATISTICS" value="1"/>
+  <arg name="world_name" default="example"/>
+  <arg name="name" default="plane"/>
+  <arg name="link_name"       value="fuselage"/>
+  <arg name="sensor_prefix"   value="/world/$(arg world_name)/model/$(arg name)/link/$(arg link_name)/sensor"/>
+  <arg name="breadcrumbs" default="0"/>
+
+  <group ns="$(arg name)">
+
```

Comment:
Please add an Ignition to ROS bridge node for your IMU sensor. [Example](https://github.com/osrf/subt/blob/master/submitted_models/x3_sensor_config_5/launch/vehicle_topics.launch#L17)